lib/settings.coffee

```diff
@@ -1,8 +1,15 @@
 module.exports =
   config:
+
+    fontSize:
+      title: 'Font Size'
+      description: 'Change the UI font size. Needs to be between 10 and 20.'
+      type: 'string'
```

Comment:
Type should probably be `integer`. Also use `minimum` and `maximum` values rather than specifying in the description.